source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.mm

```diff
@@ -73,6 +73,7 @@ - (instancetype) init
              [ACRContainerRenderer getInstance],  [NSNumber numberWithInt:(int)[ACRContainerRenderer elemType]],
              [ACRColumnSetRenderer getInstance],  [NSNumber numberWithInt:(int)[ACRColumnSetRenderer elemType]],
              [ACRColumnRenderer getInstance],     [NSNumber numberWithInt:(int)[ACRColumnRenderer elemType]],
+             [ACRActionSetRenderer getInstance],   [NSNumber numberWithInt:(int)[ACRActionSetRenderer elemType]],
```

Comment:
>  [](start = 50, length = 1)

[NIT] remove one space

Reply:
thanks

---
In reply to: [270519273](https://github.com/Microsoft/AdaptiveCards/pull/2582#discussion_r270519273) [](ancestors = 270519273)